### PR TITLE
Changed user circle color to blue with low opacity.

### PR DESCRIPTION
**Pull request by LegendCK**
## Description
Please include issue number and a summary of the change.
Issue #32 

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

 ## Screenshot of successful build.
image

## Screenshots / Screen Recordings
This should be the color which i have updated.
image

### CODE REVIEW

**Comment by LegendCK** [comment]
Can you just check and tell me whether it worked. Cause I not having Google maps API key, due to which i am not able check with maps. 

If there's any changes to perform, let me know.

**Comment by Vishwajith-Shettigar** [comment]
@LegendCK also remove the black stroke.

File: app/src/main/java/com/example/nextgen/nearby/NearByFragment.kt [hunk 1]
```diff
@@ -311,7 +311,7 @@ class NearByFragment : BaseFragment(), OnMapReadyCallback, UpdateMapListener {
         .center(location)
         .radius(CIRCLE_RADIUS.toDouble())
         .strokeColor(R.color.black)
-        .fillColor(R.color.teal_200)
+        .fillColor(Color.argb(51, 72, 133, 237))
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/nearby/NearByFragment.kt]: Use this color from colors file.

**Review by Vishwajith-Shettigar — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/nextgen/nearby/NearByFragment.kt (hunk 1 above)
Comment (LegendCK) [reply app/src/main/java/com/example/nextgen/nearby/NearByFragment.kt]: @Vishwajith-Shettigar  Check now.

**Comment by Vishwajith-Shettigar** [comment]
Thanks @LegendCK .

**Comment by LegendCK** [comment]
> Thanks @LegendCK .

Happy to help.

**Pull request merged**